CHANGELOG.md

```diff
@@ -2,9 +2,12 @@
 
 ## unreleased
 
+* Create `GooglePaymentClient`
+* Migrate GooglePayment supplemental classes that originally existed in the `braintree-android` repo into this repo.
 * Bump `compileSdkVersion` and `targetSdkVersion` to API level 30
 * Breaking Changes
   * Bump `minSdkVersion` to 21.
+  * Remove `GooglePayment` static client.
```

Comment:
I wonder if this bullet point and the one that says "Create `GooglePaymentClient`" should be combined so that it's clear that they go together. It could be something like:
```suggestion
  * Replace `GooglePayment` with `GooglePaymentClient`
```